src/Microsoft.Azure.SignalRService/Scope.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalRService
+{
+    public enum Scope
+    {
+        Client, // scope for  client to connect to a hub
```

Comment:
>Client [](start = 8, length = 6)

not clear to me what it is. where is `Scope` used? maybe add it when we need.

Reply:
Used in `GenerateAccessToken` in src/Microsoft.Azure.SignalRService/ServiceManager.cs